cmd/retry.go

```diff
@@ -113,5 +116,27 @@ func newRetryTimer(unit time.Duration, cap time.Duration, jitter float64, doneCh
 
 		}
 	}()
+
+	// Start reading..
 	return attemptCh
 }
+
+// Default retry constants.
+var (
+	defaultRetryUnit = time.Second      // 1 second.
+	defaultRetryCap  = 30 * time.Second // 30 seconds.
```

Comment:
Cap is not a multiple of Unit - Unit can be '2 * time.Second' as well and as well as something else too.